server/app-engine/main.py

```diff
@@ -104,15 +104,19 @@
 @app.route('/service-alerts.pb')
 def service_alerts():
     print('/service-alerts.pb')
+
     agency = request.args.get('agency')
     print('- agency: ' + agency)
-
     if agency is None:
         return 'No agency given', 400
 
+    include_future_alerts = request.args.get("include_future_alerts") or "False"
```

Comment:
The concept of including or not including future alerts seems like a design decision that should be discussed before going to implementation. Again, I haven't see all of the changes yet.

Reply:
Happy to discuss (here are in meeting tomorrow), but I do think the use cases are pretty straightforward:
- For generating the protobuf file for wayfinding apps, it makes sense to only show alerts that are in effect right now.
- For the purpose of managing service alerts, it would be misleading to omit alerts that aren't in effect yet, so I made the call to show future ones as well.